index.bs

```diff
@@ -1483,84 +1821,50 @@ Include some non-normative information about device configurations.
   interface USBInterface {
     constructor(USBConfiguration configuration, octet interfaceNumber);
     readonly attribute octet interfaceNumber;
-    readonly attribute USBAlternateInterface alternate;
+    [SameObject] readonly attribute USBAlternateInterface alternate;
```

Comment:
Same here.

Reply:
Updated in https://github.com/WICG/webusb/pull/212/commits/1a96f493fc285da99d2b812d809c241388197beb.